packages/frontend/src/widgets/popups/success/callback-status-success/ui/SuccessCallbackPopup.module.scss

```
@@ -0,0 +1,71 @@
+@use 'src/app/styles/core/variables' as *;
+@use 'src/app/styles/core/breakpoints' as *;
+@use 'src/app/styles/core/typography' as *;
+
+.content {
+  display: flex;
+  flex-direction: column;
+  gap: 30px;
+  align-items: center;
+  margin: 60px 110px 70px;
+
+  @include bp-743 {
+    gap: 30px;
+    margin: 30px 30px 60px;
+  }
+  @include bp-479 {
+    gap: 40px;
+    margin: 40px 20px 50px;
```

У тебя расстояние от края кнопки должно быть. Там сверху и снизу 40px. (визуально не так выглядит)
```suggestion
    margin: 40px 20px;
```

---

Поправил